.github/actions/environment/action.yml

```diff
@@ -85,4 +85,30 @@ runs:
         # files from previous runs, to avoid of it need to run:
         git clean -xffd
         git submodule foreach --recursive 'git clean -xffd'
+
+        # A temporary workaround that should be removed after the corresponding
```

Comment:
Side note: Why this should be removed? I consider this as a nice practice against runner outage.

Reply:
Because when the bug is fixed there will be no need to do it.